gradle/verification-metadata.xml

```diff
@@ -664,6 +667,16 @@
             <sha256 value="cdedd4655949d142a15e788dd88908845c0d856c436b28ef3974e24627eb9d77" origin="Generated by Gradle because artifact wasn't signed"/>
          </artifact>
       </component>
+      <component group="com.gradle" name="gradle-enterprise-gradle-plugin" version="3.3.2">
```

Comment:
I adding the 3.3.2 version necessary? Did we use that anywhere?

Reply:
We added it for consistency as `3.3.2` was released yesterday but we never integrated it into the Gradle build. Should we still remove it?